src/main/kotlin/org/codefreak/codefreak/util/TarUtil.kt

```diff
@@ -281,8 +282,10 @@ object TarUtil {
     createEntryInTar(name, outputStream, EntryType.FILE)
   }
 
-  fun normalizeFileName(name: String) = FileUtil.sanitizeName(name).withoutTrailingSlash()
-  fun normalizeDirectoryName(name: String) = FileUtil.sanitizeName(name).withTrailingSlash()
+  fun normalizeFileName(name: String) = FilenameUtils.separatorsToUnix(FileUtil.normalizePath(name)).withoutTrailingSlash().trimStart('/')
+  fun normalizeDirectoryName(name: String) = FilenameUtils.separatorsToUnix(FileUtil.normalizePath(name)).withTrailingSlash().trimStart('/')
```

Comment:
I think we should explain here that directories are with and files are without trailing slashes